addons/parachute/functions/fnc_handleFailureChance.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+/*
+ * Author: JoramD
+ * Handles percentage chance parachute failure.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, vehicle player] call FUNC(handleFailureChance)
+ *
+ * Public: No
+ */
+
+params ["_unit", "_vehicle"];
+
+if !(GVAR(failureChance) > 0 || {_vehicle isKindOf "ParachuteBase"}) exitWith {};
```

Comment:
```suggestion
if !(GVAR(failureChance) > 0 && {_vehicle isKindOf "ParachuteBase"}) exitWith {};
```

Reply:
imho this is more readable
```suggestion
if (GVAR(failureChance) == 0 || {!(_vehicle isKindOf "ParachuteBase")}) exitWith {};
```

Reply:
De Morgan says you need to change or to and.